test/integration/commons.js

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2020 Mozilla Foundation
```

Comment:
Let's rename this file to `test_utils.js` to match the same kind of file we already have for the unit tests. It's also a bit more descriptive that it contains utility functions for the tests themselves.